app/facades/explore_facade.rb

```diff
@@ -0,0 +1,16 @@
+class ExploreFacade
+  attr_reader :scores
+
+  def initialize(current_user)
+    @users =  User.where.not(id: current_user.id).where.not(flat_id: nil) #Need to make this pull location sorted list.
+    @scores = get_scores
+  end
```

Comment:
Deliberately trying to pass users other than current user who have a flat_id. This'll need replaced with a list of users from the microservice eventually.

Reply:
Right that should be an easy switch over when we're ready